src/Xamarin.Android.Tools.AndroidSdk/AndroidSdkInfo.cs

```diff
@@ -185,5 +185,18 @@ public static void DetectAndSetPreferredJavaSdkPathToLatest (Action<TraceLevel,
 			var sdk = CreateSdk (logger);
 			sdk.SetPreferredJavaSdkPath (latestJdk.HomePath);
 		}
+
+		public static string GetPreferredAndroidToolsPath(string androidSdkPath)
+		{
+			var toolsDir = Path.Combine (androidSdkPath, "cmdline-tools");
```

Comment:
@JonDouglas  so this method needs to also check for the `tools` if `cmdline-tools` does not exist. Currently if it doesn't exist it will return `cmdline-tools` as per line 193

Reply:
@DmitriyKirakosyan ^

Reply:
@dellis1972 Sounds absolutely reasonable, but I was thinking if supporting old 'tools' could potentially cause any additional issues. If we fallback to it, then Android SDK detection and validation would succeed. But do we really want an SDK using an obsolete component to be valid?

Reply:
Perhaps avoiding breaking users setup is more important, I agree.
I'll add a fallback.